packages/forms-web-app/src/validators/final-comment/final-comment.js

```diff
@@ -0,0 +1,25 @@
+const { body } = require('express-validator');
+const {
+	rule: doesNotIncludeSensitiveInformationRule
+} = require('../common/does-not-include-sensitive-information');
+const { numberToLocaleString } = require('../../lib/number-functions');
+
+const maxCharacters = 32500;
```

Comment:
could/should this go in config file?

Reply:
Just checking, would this be best placed in `src/configuration/config.js`?  And if so should we be pulling the value from an environment variable?

Reply:
you could put it there, and do a `key: environment variable || default value` sort of thing, with the default being 32500?

Reply:
Done!